src/MoBi.Presentation/Presenter/SelectNeighborPathPresenter.cs

```diff
@@ -47,9 +47,7 @@ private void onSelectedContainerPathChanged(IEntity entity, ObjectPath container
          if (container.Mode != ContainerMode.Physical)
             return;
 
-         var parentPath = container.RootContainer.ParentPath?.Clone<ObjectPath>() ?? new ObjectPath();
```

Comment:
We were adding the patent path explicitly . No need to since we assume the path was already created as expected

Reply:
now I just create the absolute path right in the selection so that we are in control